scout/server/extensions/loqus_extension.py

```diff
@@ -235,7 +235,7 @@ def get_api_loqus_variant(api_url, variant_info):
             end = variant_info["end"]
 
             sv_type = variant_info["variant_type"]
-            search_url = f"{search_url}/svs/?chrom={chrom}&end_chrom={end_chrom}&pos={pos}&end={end}&sv_type={sv_type}"
+            search_url = f"{search_url}/?chrom={chrom}&end_chrom={end_chrom}&pos={pos}&end={end}&sv_type={sv_type}"
```

Comment:
This is interesting. Obviously an error I've introduced when I've created the support for the loqus API. But how come that this works if the loqusdbapi endpoint is shaped like this: https://github.com/Clinical-Genomics/loqusdbapi/blob/3fb1830e3b8f3d04a6e638f12d454e95475e8030/loqusdbapi/main.py#L98 ??

🤔

Reply:
The search url already has the category here:
https://github.com/Clinical-Genomics/scout/blob/3995a367b4a5e56dd23e9ee172992cc74b69e6e0/scout/server/extensions/loqus_extension.py#L227

Reply:
So it was added twice, basically, once explicitly once implicitly..

Reply:
Aha! I see now!

Reply:
> Didn't we test how to retrieve SVs in #2349?
> 
> Look at the comments with the validations.. 🤔

Yes, and I also tested it in https://github.com/Clinical-Genomics/loqusdbapi/issues/19, where it failed first, then seemed to work. I think I am missing something about the connection session longevity: I guess both extensions must exist at startup, but perhaps we choose a little too early on a page which one to use?